.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature_Request
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the Feature you want to end**
```

Comment:
This line doesn't make sense it should be something like 'Explain and describe the feature you wish to be added'